backend/src/tasks/checkUserExpiration.ts

```diff
@@ -0,0 +1,129 @@
+import { Handler } from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import { connectToDatabase, User, UserType } from '../models';
+import { subDays, formatISO } from 'date-fns';
+import { sendEmail } from '../api/helpers';
+import { getRepository } from 'typeorm';
+
+const cognito = new AWS.CognitoIdentityServiceProvider();
+const userPoolId = process.env.REACT_APP_USER_POOL_ID!;
```

Comment:
This is failing in the tests. I confirmed it does exist in backend/env.yml

Maybe this has to be within the handler?

<img width="1201" alt="Screenshot 2024-02-08 at 9 31 48 AM" src="https://github.com/cisagov/crossfeed/assets/79927030/206c4ffb-b1e3-4fe6-95b7-855790cbb8a3">